tests/scr-readingsystem-support_iframe/OPS/package.opf

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Scripting</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:date>2021-12-10</dc:date>
+    <dc:description>Checks whether the reading system implements the epubReadingSystem object in container-constraint document.</dc:description>
```

Comment:
Should this be container-constrained? Maybe this is a term of art, but in normal English constraint isn't an adjective or adverb. Also needs either "a" or "documents" plural.

Also, I think this should say something specifically about an iframe, like "in a container-constrained document (e.g. an iframe)."

Reply:
Yep, my mistake. Done.